app/assets/javascripts/live_search.js

```diff
@@ -187,40 +226,41 @@
    * rewrite the appropriate tracking data attribute to delineate the group and document index.
    */
   LiveSearch.prototype.indexTrackingData = function indexTrackingData () {
-    var $groupEls = $('.filtered-results__group')
+    var $groupEls = document.querySelectorAll('.filtered-results__group')
     if ($groupEls.length > 0) {
```

Comment:
This conditional seems redundant, as the following lines will just not execute anything if the query returns an empty NodeList.

Reply:
Good point, will change.